docs/tasks/configure-pod-container/tcp-liveness-readiness.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goproxy
+  labels:
+    app: goproxy
+spec:
+  containers:
+  - name: goproxy
+    image: gcr.io/google_containers/goproxy:0.1
+    ports:
+    - containerPort: 8080
+    readinessProbe:
+      tcpSocket:
+        port: 8080
+      initialDelaySeconds: 5
+      periodSeconds: 10
+    livenessProbe:
+      tcpSocket:
```

Comment:
I was originally trying to keep this section dedicated to TCP probes to complement the earlier `exec` and `httpGet` examples. I'd be concerned that adding too many types of probes in a single example could get confusing. Do you think it would work to add a note here that a single container can use all kinds of probes at the same time? If not, I can definitely work on reworking the example to include multiple types of checks.